instana/util/traceutils.py

```diff
@@ -30,3 +30,16 @@ def get_active_tracer():
         # Do not try to log this with instana, as there is no active tracer and there will be an infinite loop at least
         # for PY2
         return None
+
+
+def get_tracer_tuple():
+    active_tracer = get_active_tracer()
+    if get_active_tracer():
```

Comment:
Since we have already defined `active_tracer = get_active_tracer()`
```suggestion
    if active_tracer:
```

Reply:
Done